meta.yaml

```diff
@@ -0,0 +1,42 @@
+package:
+  name: "pytest-playwright"
+  version: "{{ environ.get('GIT_DESCRIBE_TAG') }}"
+
+source:
+  path: .
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - setuptools-scm
+    - pip
+  run:
+    - python >=3.7
+    - playwright >=1.18
+    - pytest >=6.2.4,<8.0.0
+    - pytest-base-url >=1.0.0,<3.0.0
+    - python-slugify >=6.0.0,<9.0.0
+
+test:
+  imports:
+    - pytest_playwright
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/microsoft/playwright-pytest
+  summary: A pytest wrapper with fixtures for Playwright to automate web browsers
+  license: Apache-2.0
+  license_file: LICENSE
+
+extra:
```

Comment:
sounds like we don't need this since we don't push to conda forge, we push to the microsoft channel instead.

Reply:
Sure thing, I'll remove that